sql/core/src/main/scala/org/apache/spark/sql/execution/columnar/InMemoryRelation.scala

```diff
@@ -163,7 +163,7 @@ case class InMemoryRelation(
     output: Seq[Attribute],
     @transient cacheBuilder: CachedRDDBuilder,
     override val outputOrdering: Seq[SortOrder])(
-    statsOfPlanToCache: Statistics)
+    var statsOfPlanToCache: Statistics)
```

Comment:
it's just a nit but I think `var` on case class is discouraged per https://github.com/databricks/scala-style-guide#case-classes-and-immutability

Reply:
Yea, I know and I used `var` here based on `CachedRDDBuilder` https://github.com/apache/spark/blob/255daf03629fe18142dca751fe5c305e353befcd/sql/core/src/main/scala/org/apache/spark/sql/execution/columnar/InMemoryRelation.scala#L52

But, you're right; if we can, it's the best to do that. I'll keep looking for another solution.